<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="BlockArea" name="blockArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openDir"/>
   </widget>
   <widget class="QMenu" name="view">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="sortByExtCount"/>
    <addaction name="sortByFileSize"/>
   </widget>
   <addaction name="file"/>
   <addaction name="view"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openDir">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="sortByExtCount">
   <property name="text">
    <string>按碎片数排序</string>
   </property>
  </action>
  <action name="sortByFileSize">
   <property name="text">
    <string>按文件大小排序</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlockArea</class>
   <extends>QFrame</extends>
   <header>blockarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
